db/upgrade.php

```diff
@@ -316,5 +316,23 @@ function xmldb_customcert_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2024042210, 'mod', 'customcert');
     }
 
+    if ($oldversion < 2025062100) {
```

Comment:
This should be the value of the version for version.php before it was bumped. The reason is if we use the current date and then do an upgrade, then minor versions would have the same version as the `main` branch. This can cause issues because we may have introduced an upgrade step 2 weeks ago on just `main` for a new feature then when people upgrade to the `main` they miss the upgrade as the date they have is past that.